wiki/Announcement_messages/en.md

```diff
@@ -0,0 +1,49 @@
+---
+tags:
+  - announce
+  - announce usergroup
+  - announce user group
+---
+
+# Announcement messages
+
+![Announcement notification](img/notification.jpg "An announcement message notification")
+
+An **announcement message** is a special type of message that's intended for sending longer and formatted messages to multiple users at once. The key differences between announcement messages and regular chat messages are:
+
+- Announcement messages have a 1024 character limit instead of 450.
+- Announcement messages support Markdown[^note-images] syntax for text formatting.
+- Announcement messages are sent to all users in the specified channel.
+- Announcement messages bypass the `block private messages from people not on your friends list` setting.
+- Normal users cannot reply to announcement messages.
```

Comment:
making it less repetitive
also, 3) isn't really a difference, since regular chat messages are visible by all channel users as well
```suggestion
- The 1024 character limit instead of 450
- Markdown syntax support[^note-images] for text formatting
- Delivery to all users in the specified channel
- Ability to bypass the `block private messages from people not on your friends list` setting.
- Normal users cannot reply to announcement messages.
```

Reply:
i'll rename it to something like `Delivery to multiple users at once`